### PR TITLE
[16.0][ADD] purchase_order_owner

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
This module adds Assign Owner (owner_id) field in purchase order, whose value will be
passed to the corresponding field of the incoming picking upon order confirmation.

@qrtl

### CODE REVIEW

**Review by quannguyen1560 — APPROVED** [review]
LGTM!

File: purchase_order_owner/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module adds Assign Owner (owner_id) field in purchase order, whose value will be
+passed to the corresponding field of the incoming picking upon order confirmation.
```
Comment (yostashiro) [inline purchase_order_owner/readme/DESCRIPTION.rst:2]:
```suggestion
passed to the corresponding field of the incoming picking upon order confirmation.

This module can be useful in a business situation where you purchase products in place of a client (you seprately issue an invoice to the client and the inventory is owned by them, although you physically keep the inventory in your premise).
```

**Review by yostashiro — COMMENTED** [review]
@AungKoKoLin1997 Can you please update the readme with some business context.

**Comment by AungKoKoLin1997** [comment]
Tests are failed because of purchase_discount module. In purchase_discount module, inherit _get_stock_move_price_unit() to update `price_unit`. Odoo add a [commit](https://github.com/odoo/odoo/commit/c8aecfdd0d3de5873317da515e9f0dc507152834) to call _get_stock_move_price_unit() in the write() function. So, updating `price_unit` causes the maximum recursion depth.

**Comment by AungKoKoLin1997** [comment]
After this PR is forward to 16.0, the tests will be successful.
https://github.com/OCA/purchase-workflow/pull/1826

**Review by yostashiro — APPROVED** [review]
Code reivew and functional test. 👍

**Pull request merged**